spec/models/video_spec.rb

```diff
@@ -7,6 +7,7 @@
   it { should validate_presence_of(:title) }
   it { should validate_presence_of(:description) }
   it { should have_many(:reviews).order("created_at DESC") }
+  it { should have_many(:queue_items).order(:position) }
 
   describe "search_by_title" do
```

Comment:
Do I need to correct anything here? Rather than using  `should` , should I use the `is_expected.to` syntax instead?